js/src/data/resolvers.js

```diff
@@ -462,3 +463,35 @@ export function* getStoreCategories() {
 		);
 	}
 }
+
+/**
+ * Resolver for getting the Tours.
```

Comment:
Can we get multiple Tours or the Tour?

Reply:
No. Adjusted the wording in https://github.com/woocommerce/google-listings-and-ads/pull/1879/commits/f3d7c5b58bd367b277b36e91a9da559388cbc20e.